Flake test  TestAddRemovePodWaypoint

https://github.com/kmesh-net/kmesh/actions/runs/11285321459/job/31387912728?pr=900
```
=== RUN   TestAddRemovePodWaypoint/after
2024-10-11T02:45:41.358878Z	info	tf	=== BEGIN: Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after]' ===
=== RUN   TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity
2024-10-11T02:45:41.359113Z	info	tf	=== BEGIN: Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity]' ===
    baseline_test.go:512: 2 errors occurred:
        	* failed calling service-with-waypoint-at-service-granularity (cluster=cluster-0)->'http://10.244.1.5:18080/': call failed from service-with-waypoint-at-service-granularity (cluster=cluster-0) to http://10.244.1.5:18080/ (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        	* failed calling service-with-waypoint-at-service-granularity (cluster=cluster-0)->'http://10.244.1.5:18080/': call failed from service-with-waypoint-at-service-granularity (cluster=cluster-0) to http://10.244.1.5:18080/ (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        
2024-10-11T02:46:41.360333Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity] (1m0.001215283s)' ===
=== RUN   TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity#01
2024-10-11T02:46:41.360668Z	info	tf	=== BEGIN: Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity#01]' ===
    baseline_test.go:512: 2 errors occurred:
        	* failed calling service-with-waypoint-at-service-granularity (cluster=cluster-0)->'http://10.244.1.7:18080/': call failed from service-with-waypoint-at-service-granularity (cluster=cluster-0) to http://10.244.1.7:18080/ (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        	* failed calling service-with-waypoint-at-service-granularity (cluster=cluster-0)->'http://10.244.1.7:18080/': call failed from service-with-waypoint-at-service-granularity (cluster=cluster-0) to http://10.244.1.7:18080/ (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        
2024-10-11T02:47:41.36[1983](https://github.com/kmesh-net/kmesh/actions/runs/11285321459/job/31387912728?pr=900#step:4:1984)Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity#01] (1m0.001310965s)' ===
=== RUN   TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh
2024-10-11T02:47:41.362311Z	info	tf	=== BEGIN: Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh]' ===
    baseline_test.go:512: 2 errors occurred:
        	* failed calling enrolled-to-kmesh (cluster=cluster-0)->'http://10.244.1.5:18080': call failed from enrolled-to-kmesh (cluster=cluster-0) to http://10.244.1.5:18080 (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        	* failed calling enrolled-to-kmesh (cluster=cluster-0)->'http://10.244.1.5:18080/': call failed from enrolled-to-kmesh (cluster=cluster-0) to http://10.244.1.5:18080/ (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        
2024-10-11T02:48:41.363989Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh] (1m0.00167427s)' ===
=== RUN   TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh#01
2024-10-11T02:48:41.364342Z	info	tf	=== BEGIN: Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh#01]' ===
    baseline_test.go:512: 2 errors occurred:
        	* failed calling enrolled-to-kmesh (cluster=cluster-0)->'http://10.244.1.7:18080': call failed from enrolled-to-kmesh (cluster=cluster-0) to http://10.244.1.7:18080 (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        	* failed calling enrolled-to-kmesh (cluster=cluster-0)->'http://10.244.1.7:18080': call failed from enrolled-to-kmesh (cluster=cluster-0) to http://10.244.1.7:18080 (using http): 10 errors occurred:
        	* response[0]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[1]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[2]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[3]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[4]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[5]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[6]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[7]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[8]: X-Request-Id set, is L7 processing enabled unexpectedly?
        	* response[9]: X-Request-Id set, is L7 processing enabled unexpectedly?
        
        
[2024](https://github.com/kmesh-net/kmesh/actions/runs/11285321459/job/31387912728?pr=900#step:4:2025)-10-11T02:49:41.365715Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh#01] (1m0.001369123s)' ===
2024-10-11T02:49:41.365776Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint/after] (4m0.006895877s)' ===
2024-10-11T02:49:41.365810Z	info	tf	=== DONE (failed):  Test: '_home_runner_work_kmesh_kmesh_test_e2e[TestAddRemovePodWaypoint] (4m5.259670571s)' ===
--- FAIL: TestAddRemovePodWaypoint (248.29s)
    --- PASS: TestAddRemovePodWaypoint/before (0.05s)
        --- PASS: TestAddRemovePodWaypoint/before/from_service-with-waypoint-at-service-granularity (0.01s)
        --- PASS: TestAddRemovePodWaypoint/before/from_service-with-waypoint-at-service-granularity#01 (0.01s)
        --- PASS: TestAddRemovePodWaypoint/before/from_enrolled-to-kmesh (0.01s)
        --- PASS: TestAddRemovePodWaypoint/before/from_enrolled-to-kmesh#01 (0.01s)
    --- FAIL: TestAddRemovePodWaypoint/after (240.01s)
        --- FAIL: TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity (60.00s)
        --- FAIL: TestAddRemovePodWaypoint/after/from_service-with-waypoint-at-service-granularity#01 (60.00s)
        --- FAIL: TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh (60.00s)
        --- FAIL: TestAddRemovePodWaypoint/after/from_enrolled-to-kmesh#01 (60.00s)
=== RUN   TestRemoveAddNsOrServiceWaypoint
```

--- a/test/e2e/baseline_test.go
+++ b/test/e2e/baseline_test.go
@@ -508,7 +508,7 @@ func TestAddRemovePodWaypoint(t *testing.T) {
 				}
 				for _, dstWl := range dst.WorkloadsOrFail(t) {
 					t.NewSubTestf("from %v", src.Config().Service).Run(func(t framework.TestContext) {
-						c := IsL4()
+						c := IsL7()
 						opt := echo.CallOptions{
 							Address: dstWl.Address(),
 							Port:    echo.Port{ServicePort: ports.All().MustForName("http").WorkloadPort},

